tests_requirements/test_cases.py

```diff
@@ -50,3 +53,15 @@ def test_model_custom_requirements():
     reqs = [x.name for x in model.requirements.custom]
     assert 'test_pack_1' in reqs
     assert 'test_pack_2' in reqs
+
+
+def test_metric__callable_works(tmpdir):
```

Comment:
Probably unnecessary, but it would be nice to test if the result of the metric being computed is right.

Reply:
it is tested inside run_metric code, if it is wrong, it will exit with nonzero and test will fail

Reply:
Ok , got you